src/main/java/org/jabref/gui/autocompleter/SuggestionProviders.java

```diff
@@ -60,7 +60,7 @@ public void indexEntry(BibEntry bibEntry) {
         if (InternalBibtexFields.getFieldProperties(fieldName).contains(FieldProperty.PERSON_NAMES)) {
             return new PersonNameSuggestionProvider(fieldName);
         } else if (InternalBibtexFields.getFieldProperties(fieldName).contains(FieldProperty.SINGLE_ENTRY_LINK)) {
-            return new BibEntrySuggestionProvider();
+            return new ParsedEntryLinkSuggestionProvider();
```

Comment:
Could you comment, why the `BibEntrySuggestionProvider()` cannot be used here? Maybe just add a comment before this line.